src/shogun/neuralnets/NeuralConvolutionalLayer.h

```diff
@@ -40,6 +40,20 @@
 
 namespace shogun
 {
+
+/** Specifies the type of initialization of parameters/weights for this layer.
+ */
+enum InitializationMode
+{
+	/** Specify the standard deviation of the Gaussian Distribution at the time
+	 * of initialization of the network */
+	INIT_MANUAL=0,
+	/** Sets standard deviation to
+	 * \sqrt{\frac{2}{input\_width\times input\_height\times input\_channels}}
```

Comment:
Please use doxygen syntax for equations like here: https://github.com/shogun-toolbox/shogun/blob/develop/src/shogun/kernel/GaussianKernel.h